assets/js/index.js

```diff
@@ -9,6 +9,14 @@ new Vue({
     await this.init();
     this.setup_socket();
   },
+  watch: {
+    ["player_status.now_playing"](now_playing) {
+      const app_name = "jukebox";
+      document.title = now_playing
+        ? `⏵ ${this.player_status.now_playing_content.title} - ${app_name}`
```

Comment:
'⏵' is not displayed on my environment :(
Probably the font that is set is a problem.
Because this problem is likely to occur in other environments, please delete this character.